hoard/cli.py

```diff
@@ -8,6 +8,7 @@
 from hoard.models import Dataset
 from hoard.sources.jpal import JPAL
 from hoard.sources.lincolnLab import LincolnLab
+from hoard.sources.whoas import WHOAS
```

Comment:
Not necessary, but a suggestion would be to put these imports in `hoard/sources/__init__.py` so that here (and elsewhere) you can just do
```
from hoard.sources import JPAL, LincolnLab, WHOAS
```

Reply:
Yah, good suggestion